app/helpers/email_helper.rb

```diff
@@ -1,5 +1,9 @@
 module EmailHelper
   def current_subdomain
-    Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current 
+    if Hours.single_tenant_mode?
+      ""
+    else
+      Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current
+    end
```

Comment:
Yes I need check if application is set SIGNLE_TENANT_MODE to true and it uses subdomain ie
hourstracker.superglobalcompany.com